content/templates/content/artisanprogram.html

```diff
@@ -3,25 +3,30 @@
 {% for program in artisanprograms %}
 {% if forloop.counter0|divisibleby:2 %}
 {% if forloop.first %}
-<div class="row">
-    {% if program.img %} <!--  if image 4, 8 col -->
-    <div class="col-md-7 col-md-push-5">
-        <img class="center-block img-responsive ethics-image" src="{{ program.img.url }}">
+
+    {% if program.img %}
+<div class="text-center titlebackground" style="background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url({{ program.img.url }})
+    center 30% no-repeat; min-height:400px; ">
+    <div class="row">
+        <div class="col-md-12 text-center">
+            <h1 class="ethics-title text-center text-uppercase">{{ program.title }}</h1>
+            {% if program.text %}
+            <h4 class="white-text page-sub-title">{{ program.text | safe }}</h4>
+            {% endif %}
+        </div>
     </div>
-    <div class="col-md-5 col-md-pull-7">
-        <h1 class="ethics-title text-center text-uppercase">{{ program.title }}</h1>
-        {% if program.text %}
-        <div class="center-block program-text-right">{{ program.text | safe }}</div>
+</div>
+  {% else %}
+<div class="row">
+    <div class="col-md-12 text-center">
+        <h1 class="text-center no-img-title text-uppercase">{{ post.title }}</h1>
+        {% if post.text %}
+        <h4 class="page-sub-title">{{ post.text | safe }}</h4>
         {% endif %}
     </div>
-    {% else %}<!-- if no img center the title in the first row -->
-    <div class="col-md-12">
-        <h1 class="ethics-title text-center text-uppercase">{{ program.title }}</h1>
-    </div>
-    {% endif %}
 </div>
-<!--  end row 1 -->
-
+    {% endif %}
+    <!-- end row one -->
```

Comment:
some says `end row one` some says `end row 1`. We should probably make it consistent